pilot/pkg/serviceregistry/serviceentry/conversion.go

```diff
@@ -272,6 +272,10 @@ func convertEndpoint(service *model.Service, servicePort *networking.Port,
 			Labels:         endpoint.Labels,
 			TLSMode:        tlsMode,
 			ServiceAccount: sa,
+			// Workload entry config name is used as workload name, which will appear in metric label.
+			// After VM auto registry is introduced, workload group annotation should be used for workload name.
+			WorkloadName: wleck.name,
```

Comment:
@douglas-reid  I am using workload entry name for workload name for now. Let me know if this is expected or should I leave it empty and only fill in group annotation when it is ready.

Reply:
Talked with Doug offline, it is fine to use workload entry for now. @stevenctl just fyi, this will need some coordination with auto vm registration change. We will need to use the workload group annotation for the workload name.